x-pack/plugins/task_manager/server/lib/log_health_metrics.ts

```diff
@@ -80,9 +81,7 @@ export function logHealthMetrics(
     // This is legacy support - we used to always show this
     logger.debug(message);
     if (logLevel !== LogLevel.Debug && lastLogLevel === LogLevel.Debug) {
-      logger.warn(
-        `Detected potential performance issue with Task Manager. Set 'xpack.task_manager.monitored_stats_health_verbose_log.enabled: true' in your Kibana.yml to enable debug logging`
-      );
+      logger.debug(detectedProblemMessage);
```

Comment:
I understand why we're changing it back to debug for this PR, but we're effectively undoing [this issue](https://github.com/elastic/kibana/issues/101505) that was done for O11y of Alerting, where the intent was to try to have task manager self-introspect when it might be having problems and let the user know so they could turn on debug logging easily. Are we officially giving up on that part of O11y of Alerting because we've determined it's too noisy and too many false positives or will there be another follow up issue to try to achieve it in a less noisy way?

Reply:
Yeah, that's it in a nutshell - _it's too noisy and too many false positives_.  I'll open a new issue to track this, note some of the issues we've already seen.  Thanks!

Reply:
opened https://github.com/elastic/kibana/issues/109941 to track this